packages/slice-machine/components/FormFields/CheckboxControl.jsx

```diff
@@ -6,12 +6,13 @@ import { FormFieldCheckbox } from "./";
  * This components allows to set/unset the value of an arbitrary field via
  * checking/unchecking the box
  *
- * Field: the controlled Formik field that we want to manipulate label: a
- * function of text displayed next to the checkbox. Can be either a string or a
- * function controlledValue: the value of the field being controlled
- * setControlledValue: function to update the value of the controlled value in
- * Formik buildControlledValue: function to build the new value based on the
- * current controlled value and the state of the checkbox
+ * - "field": the controlled Formik field that we want to manipulate
```

Comment:
Also update comment format from last MR since it was on master ;)

Reply:
Ah nice, I will take the new way to do then. If I don't touch anything prettier break the comment, and it's not really readable